webapp/types/stake.ts

```diff
@@ -19,5 +19,8 @@ export type StakeProtocols = (typeof stakeProtocols)[number]
 export type StakeToken = EvmToken & {
   // EvmToken has a broad definition of "protocol", but for StakeToken let's make a
   // defined list of protocols that make a token a Stake one.
-  extensions: Omit<Extensions, 'protocol'> & { protocol: StakeProtocols }
+  extensions: Omit<Extensions, 'protocol'> & {
+    protocol: StakeProtocols
+    rewards: string[]
```

Comment:
Perhaps you can narrow the type of `rewards` for better type consistency. Let's say you define

```ts
Type Reward = 'hemi' | 'points'
```

Then you do 

```suggestion
    rewards: Reward[]
```


Whenever you refer to this type (using `Stake['extensions']['rewards']`, then you ensure that all type strings must be first defined here, and you can't add an incorrect defined type